### ISSUE TITLE
marking Classes as done

### ISSUE THREAD

**Issue by JMinyard1335** [JMinyard1335/IDEmacs#1]
This function is preforming as it should in regards to modifying the timestamps of an agenda entry. There are a few things that need to change.  Once the function is done executing it creates a separate window and leaves it open on a duplicate of the agenda view it was called from. as shown below:

![Screenshot from 2024-05-30 09-00-51](https://github.com/JMinyard1335/IDEmacs/assets/141966459/a3f8c821-df3d-41d4-9414-c6e836d01df6)
